runtime/plugins/org.jboss.tools.runtime.ui/src/org/jboss/tools/runtime/ui/internal/startup/JBossRuntimeStartup.java

```diff
@@ -36,12 +36,14 @@
 	private static final String JBOSS_RUNTIMES = "jboss-runtimes"; //$NON-NLS-1$
 	private static final String USER_HOME = "user.home"; //$NON-NLS-1$
 	private static final String JBOSS_EAP_HOME = "../../runtimes/jboss-eap"; 	// JBoss EAP home directory (relative to plugin)- <RHDS_HOME>/jbossas. //$NON-NLS-1$
+	private static final String CDK_HOME = "../../../cdk/openshift-vagrant/"; //$NON-NLS-1$

```

Comment:
why are we having such hardcoded paths ?

in final product it is 99% sure not going to be called openshift-vagrant.

why not just searching in a directory ?